www/caddy/src/opnsense/mvc/app/controllers/OPNsense/Caddy/forms/dialogAccessList.xml

```diff
@@ -3,42 +3,40 @@
         <id>accesslist.accesslistName</id>
         <label>Access List Name</label>
         <type>text</type>
-        <help><![CDATA[Enter a name for this access list.]]></help>
+        <help><![CDATA[<strong>Enter a name for this access list.</strong>]]></help>
```

Comment:
This makes no sense. We strip translations with HTML tags because we can't trust language translators to not break HTML...

Reply:
And for reference we try to remove bold markup as it just clutters the view of the user.. almost nothing should be bold except headers maybe

Reply:
I understand. I guess removing all of the markup is unavoidable then.